### PR TITLE
Added two sentences to describe that passwords are encoded

### PR DESCRIPTION
**Pull request by justineechen**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -146,6 +146,8 @@ include::finish/src/main/liberty/config/userRegistry.xml[tags=**]
 
 The registry has four users, `bob`, `alice`, `carl` and `dave`, and four groups, `Employee`, `TeamLead`, `Manager` and `PartTime`. Each user belongs to one or more groups.
 
+It is not recommended to store passwords in plain text. For simplicity reasons, the application in this guide uses XOR encoding to encode the passwords, as you can see in the `userRegistry.xml`. This passwords are encoded using the Liberty `securityUtility` command.
```
Comment (evelinec) [inline README.adoc]: "This passwords are encoded" => Should be "These passwords are encoded" or "The passwords are encoded"?

Comment (justineechen) [reply README.adoc]: I will change this to "The passwords are encoded"

**Comment by yeekangc** [comment]
@rhpeters, can you please review the text in this PR?  Thank you.

File: README.adoc [hunk 2]
```diff
@@ -146,6 +146,8 @@ include::finish/src/main/liberty/config/userRegistry.xml[tags=**]
 
 The registry has four users, `bob`, `alice`, `carl` and `dave`, and four groups, `Employee`, `TeamLead`, `Manager` and `PartTime`. Each user belongs to one or more groups.
 
+It is not recommended to store passwords in plain text. For simplicity reasons, the application in this guide uses XOR encoding to encode the passwords, as you can see in the `userRegistry.xml`. The passwords are encoded using the Liberty `securityUtility` command.
```
Comment (rhpeters) [inline README.adoc]: Suggested update: It is not recommended to store passwords in plain text. The application in this guide uses XOR encoding to encode the passwords. To encode passwords, use the Liberty `securityUtility` command.

Comment (rhpeters) [reply README.adoc]: @yeekangc, @justineechen, comment added.

Comment (justineechen) [reply README.adoc]: Thank you!

Comment (yeekangc) [reply README.adoc]: "It is not recommended to store passwords in plain text. The passwords in the `userRegistry.xml` file are encoded using the Liberty securityUtility command with XOR encoding."

What about the above?

Comment (rhpeters) [reply README.adoc]: @yeekangc, your suggestion looks good to me.

**Pull request merged**